code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -667,7 +683,7 @@ namespace WijmoProvider.Feature {
                         // Triggers the events of OnCellValueChange associated to a specific column in OS
                         this._triggerEventsFromColumn(
                             rowNumber,
-                            column.provider.binding,
+                            column.provider.uniqueId,
```

Comment:
This should be column.uniqueId. The uniqueId is our property, not the providers.